app/services/hyrax/listeners/file_metadata_listener.rb

```diff
@@ -39,7 +47,9 @@ def on_file_metadata_updated(event)
       # @param [Dry::Events::Event] event
       # @return [void]
       def on_file_uploaded(event)
-        # Run characterization
+        # Run characterization for original file only
+        return unless event[:metadata].original_file?
```

Comment:
```suggestion
        return unless event[:metadata]&.original_file?
```

`:metadata` probably SHOULD always be provided, but unhandled exceptions in listeners are very disruptive, so it's nice not to allow for `nil` errors.